src/libraries/System.Security.Cryptography.Encoding/tests/System.Security.Cryptography.Encoding.Tests.csproj

```diff
@@ -60,4 +73,7 @@
       <Link>Common\System\Memory\PointerMemoryManager.cs</Link>
     </Compile>
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="FsCheck.Xunit" Version="2.14.0" />
```

Comment:
This isn't used anywhere else in the repository, please stick with paradigms we already do.

Reply:
Not intending to merge, will remove once work is done.

Reply:
I removed the tests from the PR branch. We should however consider making this a paradigm we do do.